addons/mail/models/ir_http.py

```diff
@@ -3,18 +3,19 @@
 import odoo
 from odoo import models
 from odoo.http import request
-from odoo.addons.mail.controllers.webclient import WebclientController
 from odoo.addons.mail.tools.discuss import Store
 
 
 class IrHttp(models.AbstractModel):
     _inherit = 'ir.http'
 
-    def lazy_session_info(self):
-        res = super().lazy_session_info()
-        res["store_data"] = WebclientController._process_request(
```

Comment:
Note: importing from the controller was not taking overrides into account (discuss in particular)